app/controllers/dashboard.js

```diff
@@ -0,0 +1,59 @@
+const Departments = require('../models/departments');
+const Users = require('../models/users');
+const Messages = require('../models/messages');
+
+module.exports = {
+
+  orgIndex(req, res) {
```

Comment:
this should probably be just `organization`. Is not an index view and we know it's a dashboard because it's in the dashboard controller. Or maybe just `org` as that's more consistent right now.